frontend/src/scenes/actions/ActionStep.tsx

```diff
@@ -124,7 +124,7 @@ function Option(props: {
     const onOptionChange = (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>): void =>
         props.sendStep({
             ...props.step,
-            [props.item]: e.target.value,
+            [props.item]: e.target.value === '' ? null : e.target.value, // "" is a valid filter, we don't want it
```

Comment:
can this be simplified to `e.target.value || null` to catch all falsy values?

Reply:
Definitely! :)